docs/apis/plugintypes/tiny/index.md

```diff
@@ -0,0 +1,586 @@
+---
+title: TinyMCE Editor Plugins
+tags:
+  - editor_tiny
+  - tiny_
```

Comment:
What's the point of adding "_" at the end of this tag?

Reply:
It's the subplugin prefix. Any alternative suggestion?

Reply:
I was thinking to leave only "tiny". If you look at how tags work, you'll see the "_" is ignored: https://deploy-preview-393--moodledevdocs.netlify.app/docs/tags/tiny (so I think displaying "tiny" instead of "tiny_" would be clearer).

Reply:
Thanks - I've updated that.